### PR TITLE
Created an Indian branch and added a test file.

### PR DESCRIPTION
**Pull request by vrahatpuri**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: Indian/test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This final exercise on Udacity. Don't know what the story is about.
```
Comment (casey-collab) [inline Indian/test.txt]: Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.

Comment (casey-collab) [inline Indian/test.txt]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
It looks like this is a test pull request rather than a contribution to the story. That's fine, but I'm not going to merge it. If I've misunderstood, please re-open the pull request and let me know.

**Pull request closed without merging**